### PR TITLE
Instantiate/Upgrade improvements (closes #177)

### LINKED ISSUE #177: right click should say instantiate / upgrade
- [ ] Right clicking should say instantiate / upgrade
- [ ] The output log should tell you if you have done upgrade or instantiate
- [ ] Update the command description to include upgrade

### PR DESCRIPTION
**Pull request by Jakeeyturner**
Signed-off-by: Jake Turner <jaketurner25@live.com>

### CODE REVIEW

File: client/test/fabric/FabricConnection.test.ts [hunk 1]
```diff
@@ -268,9 +271,12 @@ describe('FabricConnection', () => {
                 args: ['arg1']
             });
             responsePayload.toString().should.equal('payload response buffer');
+            outputSpy.should.have.been.calledWith("Instantiating with function: 'instantiate' and arguments: 'arg1'");
         });
 
         it('should upgrade if already instantiated', async () => {
+            const output: VSCodeOutputAdapter = VSCodeOutputAdapter.instance();
+            const outputSpy = mySandBox.spy(output, 'log');
```
Comment (sstone1) [inline client/test/fabric/FabricConnection.test.ts]: y u no type? :trollface:

File: client/test/fabric/FabricConnection.test.ts [hunk 2]
```diff
@@ -259,6 +260,8 @@ describe('FabricConnection', () => {
         });
 
         it('should instantiate a chaincode', async () => {
+            const output: VSCodeOutputAdapter = VSCodeOutputAdapter.instance();
+            const outputSpy = mySandBox.spy(output, 'log');
```
Comment (sstone1) [inline client/test/fabric/FabricConnection.test.ts]: y u no type? :trollface:

Comment (Jakeeyturner) [reply client/test/fabric/FabricConnection.test.ts]: Done!

File: client/test/fabric/FabricConnection.test.ts (hunk 1 above)
Comment (Jakeeyturner) [reply client/test/fabric/FabricConnection.test.ts]: Done!

**Pull request merged**